src/point-cloud-octree-geometry-node.ts

```diff
@@ -187,10 +187,24 @@ export class PointCloudOctreeGeometryNode extends EventDispatcher implements IPo
       return Promise.resolve();
     }
 
-    return Promise.resolve(this.pcoGeometry.loader.getUrl(this.getHierarchyUrl()))
-      .then(url => this.pcoGeometry.xhrRequest(url, { mode: 'cors' }))
-      .then(res => res.arrayBuffer())
-      .then(data => this.loadHierarchy(this, data));
+    try {
+      return Promise.resolve(this.pcoGeometry.loader.getUrl(this.getHierarchyUrl()))
```

Comment:
I think it would make sense to extract the method (the basic check) to the utils class and chain that ( in all places).

Something like 
```
Promise.resolve(....)
.then(url.....)
.then(res => handleFailedRequest)
.then(res => res.arrayBuffer())
.....

```
And `handlFailedRequest` in utils

could do 
```
export async function handleFailedRequest(response: Response): Promise<Response> {
if (response != 200) {
 throw Error('Response error')
}
return response;
}